napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -129,6 +133,9 @@ def _create_process(
                 "CONDA_PINNED_PACKAGES",
                 f"napari={napari_version}{system_pins}",
             )
+            env.insert(
+                "PIP_USER_AGENT_USER_DATA", f"napari=conda-{napari_version}"
```

Comment:
I don't think we are using the pip interoperability features of conda/mamba at all, so this won't have any effect. If you want to mark the scenarios where we call `pip` from a conda/mamba/bundled installation, I think Talley's suggestion will be enough.

Reply:
This is only for pip, at the moment we are only consuming telemetry from pip. We can revisit if we are ready to consume from other data sources

Reply:
In that we case we don't need this line. This block is for `installer == mamba`, so pip won't be used here. 

Or are we recording the env vars anyway?

Reply:
I can drop it. I somehow thought this would capture if people were in the bundle running pip through console, brainfart!